bundles/org.openhab.binding.smhi/src/main/java/org/openhab/binding/smhi/internal/PointOutOfBoundsException.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.binding.smhi.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * @author Anders Alfredsson - Initial contribution
+ */
+@NonNullByDefault
+public class PointOutOfBoundsException extends Exception {
+
+    private static final long serialVersionUID = 546566512L;
+
+    public PointOutOfBoundsException() {
```

Comment:
This constructor would still be there if you remove this code, the same result less code.